first_year_scraper/requirements_pip.txt

```diff
@@ -0,0 +1,44 @@
+camelot-py
```

Comment:
@iakshat Can you please help me understand where all the requirements mentioned in the file are being used? I think, by accident, you have included all the installed python packages. I could be wrong though.

Reply:
if you're talking about this specific one, it's being used. That load of dependencies may be because of many many dependencies of camelot that were installed when I tried adding it to the env. It's library I'm using to parse the pdf, though, I'll recheck if there's some mistake.

Reply:
Oh! Cool. Just recheck once to be sure 😀

Reply:
I've checked it and have updated it in just one requirements.txt as suggested by you. It still has many dependencies maybe because of camelot.